packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -18,7 +19,7 @@ export interface TabProps extends HTMLProps<HTMLAnchorElement> {
   onChangeTab?: (event?: React.MouseEvent<HTMLAnchorElement>) => void;
   /** A number rendered next to the text. Usually used to display the number of items in a tab's view. */
   counter?: number | null;
-  suffix?: () => JSX.Element;
+  suffix?: string;
```

Comment:
No, it's part of the same feature which hasn't been released yet.

Reply:
What is the reasoning behind this change? For me it feels like the component is more extensible if you keep a function but maybe change it to:

```suggestion
  suffix?: () => React.ReactNode;
```

Then you don't need to move the `ProBadge` to `grafana-ui` and can keep that within the core project which feels like the place for it since, I'm guessing here, that we don't want that component to be used by external plugins/projects.

Reply:
> I'm guessing here, that we don't want that component to be used by external plugins/projects.

I was actually thinking that maybe some other Enterprise projects could use this component in the future.

Reply:
OK ok, then it makes sense 👍🏻